### PR TITLE
Receietab

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/um3bhyhxylmlrcsaiom5qgmwgy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1899)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/y73nxcruclwtunw3lvgyzo3eem?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1900)

File: screen/wallets/PaymentCodesList.tsx [hunk 1]
```diff
@@ -145,14 +145,13 @@ export default function PaymentCodesList() {
...
-    // @ts-ignore idk how to fix
     navigation.navigate('SendDetailsRoot', {
       screen: 'SendDetails',
       params: {
-        memo: '',
-        address: pc,
         walletID,
+        address: pc,
       },
+      merge: true,
```
Comment (Overtorment) [inline screen/wallets/PaymentCodesList.tsx:154]: i'd suggest a slightly different workflow.

when this screen is called from SendDetails screen, pass a parameter that will mean that any tap on any contact will lead to `_navigateToSend()`. maybe open in a modal..?


and if the screen is just opened, then everything works as before - with a popup that offers to rename, etc.
place button that leads to contacts on WalletDetails screen (as it was originally)

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/z45ubh2o3bqitq4zbbwl63hsha?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1912)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rqcqzud4qx4tzflpil66fehgei?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1916)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/i4mmjh24mj6fhcdkruhxocewde?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1922)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — CHANGES_REQUESTED** [review]
a bunch of tests failed

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Review by Overtorment — CHANGES_REQUESTED** [review]
whoa a lot of tests failed

**Comment by marcosrdz** [comment]
Yup. This relies on a another PR being merged

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2kc67oi2he7oxdixyv6lbu7e6e?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1961)

File: navigation/DetailViewStackParamList.ts [hunk 2]
```diff
@@ -30,7 +31,26 @@ export type DetailViewStackParamList = {
...
+      unit?: BitcoinUnit;
+      noRbf?: boolean;
+      launchedBy?: string;
+      isEditable?: boolean;
+      uri?: string;
+      addRecipientParams?: {
+        address?: string;
+        amount?: number;
+        memo?: string;
+      };
```
Comment (Overtorment) [inline navigation/DetailViewStackParamList.ts:50]: why this whole block is duplicated here and in `navigation/SendDetailsStackParamList.ts` ?
or are they different?

if they are the same, is there a way to unify them?

**Review by Overtorment — APPROVED** [review]

File: navigation/DetailViewStackParamList.ts (hunk 2 above)
Comment (Overtorment) [reply navigation/DetailViewStackParamList.ts:50]: @marcosrdz

Comment (marcosrdz) [reply navigation/DetailViewStackParamList.ts:50]: One is for a stack. The other is for the screen directly. 

Meaning..one is for when its a modal and the other is when its directly to the screen

Comment (Overtorment) [reply navigation/DetailViewStackParamList.ts:50]: so its ok to have them both? can we unify somehow?

Comment (marcosrdz) [reply navigation/DetailViewStackParamList.ts:50]: Not that im aware of as they are different parameters. I could be wrong. Perhaps @limpbrains knows

Comment (marcosrdz) [reply navigation/DetailViewStackParamList.ts]: Done

**Pull request merged**